projects/observability/src/shared/components/table/data-cell/exit-calls/exit-calls-table-cell-renderer.component.scss

```diff
@@ -0,0 +1,31 @@
+@import 'color-palette';
+@import 'font';
+
+.exit-calls-count {
+  @include body-1-regular($gray-7);
+}
+
+.api-callee-name-count {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 2px;
+
+  .api-callee-name {
+    @include ellipsis-overflow();
+    max-width: 200px;
+    font-size: 13.5px;
```

Comment:
Is this a font size from a mock (if so, could you slack it to me?) our fonts should all be coming from mixins which include all the various font styles in the design system, but I don't recall any at fractional pixels.

Reply:
Yeah, @itssharmasandeep usually the mocks have font mixin name written next to it. If not, we need to ask ux to use one of the existing ones.